docs/_layouts/default.html

```diff
@@ -44,6 +44,47 @@
             rel="stylesheet"
             href="{{ '/css/docsearch.css' | relative_url }}"
         />
+        <!-- Newsletter form -->
```

Comment:
You also need to make this same change in `docs/documentation.html`, otherwise if you access the `documentation` page and then the `Community` page, the layout is broken.

Reply:
Done